src/iden3comm/handlers/payment.ts

```diff
@@ -422,6 +426,7 @@ export class PaymentHandler
    * @inheritdoc IPaymentHandler#handlePayment
    */
   async handlePayment(payment: PaymentMessage, params: PaymentHandlerOptions) {
+    verifyExpiresTime(payment);
```

Comment:
maybe it worth adding  verifyExpiresTime(payment) to general MessageHandler or handle function of each handler (not each specifc handleMethod). I feel it is generic verification

Reply:
It is, but we have some public functions not from general handler like `handleAuthorizationRequest` and it won't be hanled in general one

Reply:
So i tried to put verification in handler at the place, where it will be called from general `handle` and from public function like `handleAuthorizationRequest `